onmt/modules/conv_multi_step_attention.py

```diff
@@ -46,22 +45,12 @@ def forward(self, base_target_emb, input_from_dec, encoder_out_top,
                 which is the combination of base emb and top output of encode
         """
 
-        # checks
-        # batch, channel, height, width = base_target_emb.size()
```

Comment:
Maybe document the dimensions of `base_target_emb` and other tensors in the docstring, just to keep it somewhere explicitly.